crates/rpc/rpc/src/eth/api/mod.rs

```diff
@@ -472,7 +472,7 @@ struct EthApiInner<Provider, Pool, Network, EvmConfig> {
     /// An interface to interact with the network
     network: Network,
     /// All configured Signers
-    signers: Vec<Box<dyn EthSigner>>,
+    signers: RwLock<Vec<Box<dyn EthSigner>>>,
```

Comment:
now that we made EthSigner Clone, I'd rather use parking_lot::RwLock here

this also means that we don't need to make `async accounts`